cache/cache.go

```diff
@@ -188,17 +216,44 @@ func (c *Cache) Close() {
 	c.waitGroup.Wait()
 }
 
-func serveWatchEvents(ctx context.Context, watchCtx *watchCtx) {
+func getAndWatchRetry(ctx context.Context, watchCtx *watchCtx) {
 	backoff := watchCtx.backoffStart
 	for {
+		var startRev int64
+		if resp, err := watchCtx.cache.kv.Get(ctx, watchCtx.cache.prefix, clientv3.WithPrefix()); err == nil {
+			watchCtx.cache.store.Set(resp.Kvs, resp.Header.Revision)
```

Comment:
This method should not be `Set`. `Set` implies single key. Please use name like `Reset`.

Reply:
Or `Restore`